cis/gcp/compute/gcp-cis-4.2-compute-block-project-wide-ssh-keys-enabled-for-vm-instances/test/gcp-cis-4.2-compute-block-project-wide-ssh-keys-enabled-for-vm-instances/failure-null-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This Sentinel configuration file should not be required if you implement my suggestion.